include/h2o.h

```diff
@@ -2240,6 +2257,7 @@ COMPUTE_DURATION(request_total_time, &req->timestamps.request_begin_at, &req->pr
 COMPUTE_DURATION(process_time, &req->processed_at.at, &req->timestamps.response_start_at);
 COMPUTE_DURATION(response_time, &req->timestamps.response_start_at, &req->timestamps.response_end_at);
 COMPUTE_DURATION(duration, &req->timestamps.request_begin_at, &req->timestamps.response_end_at);
+COMPUTE_DURATION(total, &req->timestamps.request_begin_at, &req->timestamps.response_end_at);
```

Comment:
I do not think that we need both `duration` and `total` as function definitions, since we do not need to guarantee source-level compatibility.

I would suggest either a) remove `duration` and rename `total` to `total_time` or b) remove `total`.

Reply:
Addressed at https://github.com/h2o/h2o/pull/1646/commits/cd6e57c135303667b149c199ba43d7e220bdc14a, but one thing I'm a bit wondering is about duration-stats, which still outputs only `duration` field. Which is the best choice?

1. Keep it as-is (total-time is not needed)
2. Add `total` (but it's redundant)
3. Remove `duration` and add `total` (of cause it may break backward compat)